nemo/collections/asr/data/audio_to_text.py

```diff
@@ -78,8 +89,11 @@ def _speech_collate_fn(batch, pad_id):
         audio_signal, audio_lengths = None, None
     tokens = torch.stack(tokens)
     tokens_lengths = torch.stack(tokens_lengths)
-
-    return audio_signal, audio_lengths, tokens, tokens_lengths
+    if sample_ids is None:
+        return audio_signal, audio_lengths, tokens, tokens_lengths
+    else:
+        sample_ids = torch.IntTensor(sample_ids)
```

Comment:
Use torch.tensor() as recommened in torch instead of IntTensor

Reply:
done.